.github/CODEOWNERS

```diff
@@ -22,4 +22,4 @@
 
 
 # for all files:
-* @adarsh-dell @AkshaySainiDell @francis-nijay @harishp8889 @harshitap26 @niranjan-n1 @prablr79 @satyakonduri @spriya-m @suryagupta4 @gilltaran @kenivi1
```

Comment:
Hi @lukeatdell Can you please follow the order/pattern of alphabets?
Please add a comment (mentioning email of respective owner )after line no. 12 and remove @spriya-m (# Shanmugapriya M (spriya-m)) from this file?
Thanks

Reply:
Codeowner file updated. Thanks.